compiler/src/regalloc.ml

```diff
@@ -743,6 +743,17 @@ let two_phase_coloring
   let schedule =
     if !Glob_options.lazy_regalloc then lazy_scheduling variables a
     else schedule_coloring size variables cnf a in
+  (* Give a specific error message if the bank is empty: there is no way the
+     variables can be allocated. We pick one of the variables to illustrate
+     the error message. *)
+  begin match schedule, registers with
```

Comment:
There is no point in computing a schedule. You may pick any `x` in the `variables` hash table (e.g. using `to_seq` and then `uncons`).